src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import Memes from './components/memes/memes';
+import MemesDetails from './components/memes/MemesDetails';
+
+function App() {
+  return (
+    <>
+      <main className="main__app">
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Memes />} />
+          <Route path="/memesDetail/:memesId" element={<MemesDetails />} />
+        </Routes>
+      </main>
```

Comment:
- [ ] The elements overlap on the right side, kindly update it 
https://www.awesomescreenshot.com/video/17911217?key=df83792dc63d7dcd339758b111fa3809